api/rest/coordinator_test.go

```diff
@@ -25,6 +25,8 @@ func TestCoordinator_CreatorShouldCreateAbundleAndReturnUpdateChan(t *testing.T)
 
 	ctx := context.TODO()
 
+	localBundleID := uuid.New()
```

Comment:
Nitpicking: For the tests we wouldn't even need a UUID except we actually checked the format and rejected non-UUIDv4 IDs.

Reply:
Are we using github.com/google/uuid only for tests? I think we don't need that dependency at this step.